### PR TITLE
ZIM-34 Tests for subclasses of ControlOp

### PR DESCRIPTION
**Pull request by cgeisser**

### CODE REVIEW

File: store/src/java-test/com/zimbra/cs/redolog/op/CheckpointTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CheckpointTest {
+    private Checkpoint op;
+
+    @BeforeClass
+    public static void init() throws Exception {
+        MailboxTestUtil.initServer();
```
Comment (jmif) [inline store/src/java-test/com/zimbra/cs/redolog/op/CheckpointTest.java]: We should extract this into its own Rule if we're going to be initing all of our tests this way.  May also come in handy later if we have to change the way the init process works with our fancy new HA setup.

Comment (jmif) [inline store/src/java-test/com/zimbra/cs/redolog/op/CheckpointTest.java]: Only an idea - seems out of scope for this PR.

File: store/src/java/com/zimbra/cs/redolog/op/RedoableOp.java [hunk 2]
```diff
@@ -73,7 +73,7 @@
     private int mMailboxId;
     private RedoLogManager mRedoLogMgr;
     private boolean mUnloggedReplay;  // true if redo of this op is not redo-logged
-    RedoCommitCallback mCommitCallback;
+    protected RedoCommitCallback mCommitCallback;
```
Comment (jmif) [inline store/src/java/com/zimbra/cs/redolog/op/RedoableOp.java]: Should this be private not protected?  Is it writeable from children?

Comment (cgeisser) [inline store/src/java/com/zimbra/cs/redolog/op/RedoableOp.java]: Yeah, CommitTxn uses it, unfortunately.
